### ISSUE TITLE
Unexpected Drop of Node 16 Support

### ISSUE THREAD

**Issue by aaronhargrove-grub** [contentful/contentful-export#1526]
Pull #1503 dropped node 16 support by updating to  `bfj ^8.0.0`. Would it be possible to lock that dependency off until the package officially drops node 16 support in a major update?

**Comment by marcolink** [comment]
Hi @aaronhargrove-grub 👋 
We are about to release the next major version with node@18 (LTS) being the min version.

**Comment by aaronhargrove-grub** [comment]
Right, but shouldn't that be done under a major release. Since it was done in a patch it breaks everyone on the semver that previously supported the older versions
